hof.settings.json

```diff
@@ -1,11 +1,12 @@
 {
-  "theme": "govuk",
+  "appName": "UKVI Complaints",
+  "theme": "govUK",
   "routes": [
     "./apps/ukvi-complaints"
   ],
-  "views": "./apps/ukvi-complaints/views",
+  "views": ["./apps/ukvi-complaints/views"],
   "session": {
-    "name": "hod.sid",
+    "name": "ukvi-complaints.hof.sid",
```

Comment:
On the /cookies page the session cookies is listed as hod.sid - will need to be updated

Reply:
ah...that is a good point. It may be the framework needs to be updated to replace the name. Whoops this will likely be the case in all our services. Hadn't even thought of that. Good spot. Maybe I can roll out a patch update that if this is set it uses the name to populate the page. I think from a compliancy thing it still ends in hof.sid so should be obvious